bigquery/value.go

```diff
@@ -786,6 +800,14 @@ const (
 
 	// NumericScaleDigits is the maximum number of digits after the decimal point in a NUMERIC value.
 	NumericScaleDigits = 9
+
+	// BigNumericPrecisionDigits is the maximum number of full digits in a BIGNUMERIC value.
+	// BigNumeric does support a partial digit beyond this.
```

Comment:
What does a partial digit mean?

Reply:
A nuance of the internal representation.  Removed reference to this, as the bit representation isn't relevant to most users.